session_3/blogspot/app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,86 @@
+class ArticlesController < ApplicationController
+  load_and_authorize_resource
+
+  before_action :set_article, only: [:show, :edit, :update, :destroy]
+
+  def article_params
+    params.require(:article).permit(:title, :topic, :tags, :content, :public)
+  end
+
+  # GET /articles or /articles.json
+  def index
+    @articles = Article.all
+    if !current_user
```

Comment:
This means that everything I go to the articles index page, my limit of five articles is reset.

Reply:
So I'll add another check to verify that the `sessions[:private_access]` exists or not

Reply:
Oh! I saw it just now, so I have to add an ability check to manage this